backend/server/websocket_manager.py

```
@@ -92,24 +95,25 @@ async def disconnect(self, websocket: WebSocket):
             # Still try to close the connection if possible
             try:
                 await websocket.close()
-            except:
+            except Exception:
                 pass  # If this fails too, there's nothing more we can do
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)